pyjokes/jokes_en.py

```diff
@@ -66,6 +66,10 @@
     "Knock knock ... Who's there? ... Data integrity ... Data📠✹$%@‽📠⁂#%*❾%🕞 ⑦⬜%@‽^📠☝📠✝!#^",
     "QA Engineer walks into a bar. Orders a beer. Orders 0 beers. Orders 999999999 beers. Orders a lizard. Orders -1 beers. Orders a sfdeljknesv.",
     "A product manager walks into a bar, asks for drink. Bartender says no, but will consider adding later.",
+    "How do you generate a random string? Put a first year CS student in front of VIM and tell him to save and exit.",
```

Comment:
The only problem is with the `him`, I suppose we could make this joke gender neutral just changing the verb. Instead of `tell him`, it could be something along: 'Ask a first year CS student to use VIM and to save and exit.'